src/image/ExposureInfo.cc

```diff
@@ -235,8 +237,8 @@ ExposureInfo::FitsWriteData ExposureInfo::_startWriteFits(lsst::geom::Point2I co
     /*
      * We need to define these keywords properly! XXX
      */
-    data.metadata->set("FLUXMAG0", getCalib()->getFluxMag0().first);
-    data.metadata->set("FLUXMAG0ERR", getCalib()->getFluxMag0().second);
+    // data.metadata->set("FLUXMAG0", getPhotoCalib()->getFluxMag0().first);
+    // data.metadata->set("FLUXMAG0ERR", getPhotoCalib()->getFluxMag0().second);
```

Comment:
Will these be defined properly?  I assume nothing currently depends on these values?

Reply:
Oh yeah, those. They can and should be deleted. The comment there is very long standing, and now irrelevant.